Assets/PatchKit Patcher/Resources/Languages/en.json

```diff
@@ -0,0 +1,57 @@
+{
+  "adding_new_files_0": "Adding new files...",
+  "adding_new_files_1": "Adding new files ({0}/{1})...",
+  "allocating_disk_space": "Allocating disk space...",
+  "applying_diffs_0": "Applying diffs...",
+  "applying_diffs_1": "Applying diffs ({0}/{1})...",
+  "applying_fixes": "Applying fixes...",
+  "cancel": "cancel",
+  "changelog": "Changelog",
+  "check_for_updates": "Check for updates",
+  "checking_version_integrity": "Checking version integrity...",
+  "confirm": "Confirm",
+  "dowload_speed_d": "day",
+  "dowload_speed_h": "hour",
+  "dowload_speed_kb": " KB/sec.",
+  "dowload_speed_m": "minute",
+  "dowload_speed_mb": " MB/sec.",
+  "dowload_speed_moment": "a moment",
+  "dowload_speed_s": "second",
+  "dowload_status_0": "Downloading package...",
+  "dowload_status_1": "Stalled...",
+  "downloading_fixes": "Downloading fixes...",
+  "enter_code": "Enter code",
+  "error": "An error has occured!",
+  "game_changelog": "GAME CHANGELOG",
+  "install": "Install",
+  "installing_0": "Installing...",
+  "installing_1": "Installing ({0}/{1})...",
+  "license": "Please entry license key",
+  "license_blocked": "Blocked license.",
+  "license_error": "Entred code is not valid!",
+  "license_invalid": "Invalid license.",
+  "license_service": "Service is unavailable.",
+  "no_internet_connection": "Please check your internet connection.",
```

Comment:
Please rename to `"please_check_your_internet_connection"`.

Reply:
done